src/test/groovy/com/palantir/gradle/javadist/JavaDistributionPluginTests.groovy

```diff
@@ -308,6 +307,52 @@ class JavaDistributionPluginTests extends Specification {
 
     }
 
+    def 'produce distribution with dependencies in services/lib'() {
+        given:
+        createUntarBuildFile(buildFile)
+        buildFile << sprintf('''
```

Comment:
I'm new to groovy so happy to learn the right way to do this, but from my understanding triple-ticked strings must be literals.  It felt clearer to use multiline strings instead of string replacement and concatenation.  Maybe there is a third way that is cleaner than both of these?